devito/operator.py

```diff
@@ -100,6 +101,9 @@ def __init__(self, nt, shape, dtype=np.float32, stencils=[],
         for d in dimensions:
             sym_undef.remove(d)
 
+        # Replace all occurences of the `time` dimension with `t`
+        self.stencils = [eqn.subs(time, t) for eqn in self.stencils]
+
```

Comment:
Ok, this still looks to work just as intended. I believe it is due to the fact that the treatment of time symbol `t` in the `Operator` is driven by a flag.